Jenkinsfile_CNP

```diff
@@ -269,6 +269,7 @@ withPipeline(type, product, component) {
     onPR {
       for (label in githubApi.getLabelsbyPattern(env.BRANCH_NAME, "pr-values:wa") ) {
         env.CAMUNDA_BASE_URL = "https://camunda-sptribs-case-api-pr-${CHANGE_ID}.preview.platform.hmcts.net"
+        env.WA_FEATURE_ENABLED = true
```

Comment:
This is already set [here](https://github.com/hmcts/sptribs-case-api/blob/dcd321e8cd8c0c30b9b582b18e7cf3b3548c42a0/charts/sptribs-case-api/values.wa.preview.template.yaml#L3) for preview